src/devices/Bmxx80/Bme680.cs

```diff
@@ -341,6 +344,40 @@ public int GetMeasurementDuration(Bme680HeaterProfile profile)
             return (int)Math.Ceiling(measDuration);
         }
 
+        /// <summary>
+        /// Performs a synchronous reading.
+        /// </summary>
+        /// <returns><see cref="Bme680ReadResult"/></returns>
+        public Bme680ReadResult Read()
+        {
+            SetPowerMode(Bme680PowerMode.Forced);
+            Thread.Sleep(GetMeasurementDuration(HeaterProfile));
+
+            TryReadTemperatureInternal(out var temperature);
+            TryReadPressureInternal(out var pressure, true);
```

Comment:
use explicit arg name for all calls which pass true/false since it's not obvious what it means

Reply:
Might be good to add that to the StyleCop analyzer if possible.